README.md

```diff
@@ -262,11 +262,11 @@ Both `Row` and `RowCollection` can load automatically other related data. Just u
 //Get the category id=34
 $category = $db->category[34];
 
-//Load the posts of this category
-$posts = $category->post;
+//Load the post of this category
+$post = $category->post;
 
```

Comment:
I think `$posts` may be correct here, because `$category->post` returns a `RowCollection` of posts (not a single post).

Reply:
Quite right!